.github/workflows/e2e-tests.yaml

```diff
@@ -142,7 +143,7 @@ jobs:
     # To prevent that, we delay the completion of the job until all namespaces
     # labeled "e2e-framework" have been finalized.
     - name: Wait for termination of E2E namespaces
-      if: always()
+      if: always() && steps.e2e.outcome != 'skipped'
```

Comment:
I added this small tweak to ensure we don't wait for e2e namespaces to terminate if tests didn't even run.